client/src/main/java/org/apache/rocketmq/client/impl/consumer/ProcessQueue.java

```diff
@@ -46,7 +46,7 @@
     private final TreeMap<Long, MessageExt> msgTreeMap = new TreeMap<Long, MessageExt>();
     private final AtomicLong msgCount = new AtomicLong();
     private final Lock lockConsume = new ReentrantLock();
-    private final TreeMap<Long, MessageExt> msgTreeMapTemp = new TreeMap<Long, MessageExt>();
+    private final TreeMap<Long, MessageExt> consumingMsgOrderlyTreeMap = new TreeMap<Long, MessageExt>();//subset of msgTreeMap, used when consume orderly
```

Comment:
It would be nice to have a Javadoc comment here. Something like,
```java
/**
* Subset of msgTreeMap used when consuming orderly.
*/
private final TreeMap<Long, MessageExt> msgOrderlyTreeMap = ...
```